scripts/DRL_wandb.py

```diff
@@ -0,0 +1,410 @@
+                policy_delay=2,
+                target_policy_noise=0.2,
+                target_noise_clip=0.5,
+                create_eval_env=False,
+                policy_kwargs=None,
+                verbose=0,
+                device='auto',
+                _init_setup_model=True)
+#--------------------------------------------------------#
+#                           Error                        #
+#--------------------------------------------------------#
+else:
+    raise RuntimeError(
+        F'Algorithm specified [{args.algorithm}] is not registered.')
+
+#--------------------------------------------------------#
+#                     Set Loggers                        #
+#--------------------------------------------------------#
+logger = Logger(folder="runs/", output_formats=[WandBOutputFormat()])
+model.set_logger(logger)
+
+
+# ---------------------------------------------------------------------------- #
+#       Calculating total training timesteps based on number of episodes       #
+# ---------------------------------------------------------------------------- #
+n_timesteps_episode = env.simulator._eplus_one_epi_len / \
+    env.simulator._eplus_run_stepsize
+timesteps = args.episodes * n_timesteps_episode - 1
+
+# ---------------------------------------------------------------------------- #
+#                                   CALLBACKS                                  #
+# ---------------------------------------------------------------------------- #
+callbacks = []
+
+# Set up Evaluation and saving best model
+if args.evaluation:
+    eval_callback = LoggerEvalCallback(
+        eval_env,
+        best_model_save_path='best_model/' + name,
+        log_path='best_model/' + name + '/',
+        eval_freq=n_timesteps_episode *
+        args.eval_freq,
+        deterministic=True,
+        render=False,
+        n_eval_episodes=args.eval_length)
+    callbacks.append(eval_callback)
+else:
+    callbacks.append(LoggerCallback(sinergym_logger=True))
+
+
+callback = CallbackList(callbacks)
+
+# ---------------------------------------------------------------------------- #
+#                                   TRAINING                                   #
+# ---------------------------------------------------------------------------- #
+model.learn(
+    total_timesteps=timesteps,
+    callback=callback,
+    log_interval=args.log_interval)
+model.save(env.simulator._env_working_dir_parent + '/' + name)
+
+# If the algorithm doesn't reset or close the environment, this script will do it in
+# order to correctly log all the simulation data (Energyplus + Sinergym
+# logs)
+if env.simulator._episode_existed:
+    env.close()
+
+# ---------------------------------------------------------------------------- #
+#                   Autodelete option if is a cloud resource                   #
+# ---------------------------------------------------------------------------- #
+if args.group_name and args.auto_delete:
```

Comment:
I have the next error:
```
Traceback (most recent call last):
  File "/home/alex/Escritorio/kad/sinergym/scripts/DRL_wandb.py", line 408, in <module>
    if args.group_name and args.auto_delete:
AttributeError: 'Namespace' object has no attribute 'group_name'
```
You should delete this part :)

Reply:
Will do!